### ISSUE TITLE
Docker.java:33-34: Start implementing the other parts of...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#3]
The puzzle `1-606fe8a1` from #1 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/605ae372d92aab25fcc7c0253a99f73c45eaf87e/src/main/java/com/amihaiemil/docker/Docker.java#L33-L34

The puzzle was created by Mihai Andronache on 28-Jan-18. 

Estimate: 30 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by amihaiemil** [comment]
@comdor hello

**Comment by comdor** [comment]
@amihaiemil Hi there! I'm only version `0.0.2`, so I can't do much yet. Follow me and [my website](http://comdor.co) for updates on the next version.

**Comment by 0crat** [comment]
Job [#3](https://github.com/amihaiemil/docker-java-api/issues/3) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#3](https://github.com/amihaiemil/docker-java-api/issues/3) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/801521347); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by llorllale** [comment]
@amihaiemil does this model look right to you?

```java
public interface Docker {
    public Swarm swarm();  //new method
}

public interface Swarm {
    public Nodes nodes();
    public Services services();
}
```

**Comment by amihaiemil** [comment]
@llorllale yeah, looks nice. But I think it's ok if you start only with interface Swarm; leave Nodes/Services for a next puzzle, considering you spent some time in the documentation

**Comment by llorllale** [comment]
@amihaiemil 

> you spent some time in the documentation

It was quick.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting on random build issue, see [this log](http://www.rultor.com/t/14195-376166113)

**Comment by 0crat** [comment]
> @0crat waiting on random build issue, see [this log](http://www.rultor.com/t/14195-376166113) ([here](https://github.com/amihaiemil/docker-java-api/issues/3#issuecomment-376203017))

@llorllale The impediment for [#3](https://github.com/amihaiemil/docker-java-api/issues/3) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
The puzzle `1-606fe8a1` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72) are still not solved.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#3](https://github.com/amihaiemil/docker-java-api/issues/3) is now out of scope

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71) are still not solved; solved: [#72](https://github.com/amihaiemil/docker-java-api/issues/72).

**Comment by 0pdd** [comment]
@0pdd 5 puzzles [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#86](https://github.com/amihaiemil/docker-java-api/issues/86) are still not solved; solved: [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72).

**Comment by 0pdd** [comment]
@0pdd 5 puzzles [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#86](https://github.com/amihaiemil/docker-java-api/issues/86), [#95](https://github.com/amihaiemil/docker-java-api/issues/95) are still not solved; solved: [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77).

**Comment by 0pdd** [comment]
@0pdd 5 puzzles [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#95](https://github.com/amihaiemil/docker-java-api/issues/95), [#96](https://github.com/amihaiemil/docker-java-api/issues/96) are still not solved; solved: [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#86](https://github.com/amihaiemil/docker-java-api/issues/86).

**Comment by 0pdd** [comment]
@0pdd 6 puzzles [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#95](https://github.com/amihaiemil/docker-java-api/issues/95), [#96](https://github.com/amihaiemil/docker-java-api/issues/96), [#98](https://github.com/amihaiemil/docker-java-api/issues/98), [#99](https://github.com/amihaiemil/docker-java-api/issues/99) are still not solved; solved: [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#86](https://github.com/amihaiemil/docker-java-api/issues/86).

**Comment by 0pdd** [comment]
@0pdd 6 puzzles [#103](https://github.com/amihaiemil/docker-java-api/issues/103), [#84](https://github.com/amihaiemil/docker-java-api/issues/84), [#85](https://github.com/amihaiemil/docker-java-api/issues/85), [#95](https://github.com/amihaiemil/docker-java-api/issues/95), [#98](https://github.com/amihaiemil/docker-java-api/issues/98), [#99](https://github.com/amihaiemil/docker-java-api/issues/99) are still not solved; solved: [#70](https://github.com/amihaiemil/docker-java-api/issues/70), [#71](https://github.com/amihaiemil/docker-java-api/issues/71), [#72](https://github.com/amihaiemil/docker-java-api/issues/72), [#77](https://github.com/amihaiemil/docker-java-api/issues/77), [#83](https://github.com/amihaiemil/docker-java-api/issues/83), [#86](https://github.com/amihaiemil/docker-java-api/issues/86), [#96](https://github.com/amihaiemil/docker-java-api/issues/96).
